app/assets/stylesheets/new_design/dossiers_table.scss

```diff
@@ -66,6 +66,11 @@
     }
   }
 
+  .updated-at-col {
+    text-align: center;
```

Comment:
Je crois que c'est pas une bonne pratique de centrer le contenu des cellules d'unt tableau (à part les boutons par ex), cf. https://speakerdeck.com/cherdarchuk/clear-off-the-table (slide "Right-align numbers")